src/main/flight/mixer.c

```diff
@@ -763,6 +765,14 @@ NOINLINE void mixTable(timeUs_t currentTimeUs, uint8_t vbatPidCompensation)
         throttle = applyThrottleLimit(throttle);
     }
 
+    // Handle yaw spin recovery - throttle is set to zero to prevent flyaway
+    // and to give the mixer full authority to stop the spin
+#ifdef USE_YAW_SPIN_RECOVERY
+    if (gyroYawSpinDetected()) {
+        throttle = 0.0f;
```

Comment:
Actually the full mixer authority is at 0.5 throttle if that is the reason behind this.